python/artm/wrapper/spec.py

```diff
@@ -94,7 +95,7 @@ def __init__(self, name, arguments, result=None, request=None):
     ),
     CallSpec(
         'ArtmDisposeBatch',
-        [('master_id', int), ('name', str)],
+        [('master_id', int), ('name', unicode if sys.version_info.major == 2 else str)],
     ),
```

Comment:
I think it's better to use `six` library here (i.e [text_type](https://pythonhosted.org/six/index.html#six.text_type)).